arm-network/2016-09-01/swagger/network.json

```diff
@@ -3985,14 +3985,12 @@
             "type": "string",
             "description": "The name of the virtual network gateway."
           },
-          {
-            "name": "parameters",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/VirtualNetworkGateway"
-            },
-            "description": "Parameters supplied to the Begin Reset Virtual Network Gateway operation through Network resource provider."
+		  { 
```

Comment:
So this is a breaking change. You are replacing the body parameter with a query parameter? This will break customers. What about previous api-versions ?
/cc @sphibbs @markcowl @johanste